### DISCUSSION TITLE
Non-Domain

### DISCUSSION THREAD

**Discussion by 26zl** [HotCakeX/Harden-Windows-Security#215] [Q&A] [answered]
Hi. Where do i find the scripts/code that makes the user account "non-domain" when running microsoft baseline security category?

**Comment by HotCakeX** [comment, marked as answer]
Hi, everything is included in the same file.
Microsoft security baselines are automatically downloaded from their official servers and run for non-domain joined PCs. 

This line is responsible for that task: https://github.com/HotCakeX/Harden-Windows-Security/blob/1ea886fb70e53953e4df0a9d78978ac8e0d5236c/Harden-Windows-Security.ps1#L1214

**Reply by 26zl** [reply]
Thank you!!! 🙏🏼🙏🏼

**Reply by HotCakeX** [reply]
You're welcome! 🙂
